src/requires.jl

```diff
@@ -0,0 +1,103 @@
+function __init__()
+  @require Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
+    function performance_profile(::PlotsBackend,
+                                 T :: Matrix{Float64},
+                                 labels :: Vector{S}=String[];
+                                 logscale :: Bool = true,
+                                 title :: AbstractString = "",
+                                 sampletol :: Float64 = 0.0,
+                                 drawtol :: Float64 = 0.0,
+                                 kwargs...) where S <: AbstractString
+      ns = size(T, 2)
+      xlabel, ylabel, labels = performance_profile_axis_labels(labels, ns, logscale; kwargs...)
+      (x_plot, y_plot, max_ratio) = performance_profile_data(T, logscale=logscale, sampletol=sampletol, drawtol=drawtol)
+      kwargs = Dict{Symbol,Any}(kwargs)
+      linestyles = pop!(kwargs, :linestyles, Symbol[])
```

Comment:
I think it would be better if `performance_profile` define the common values and calls a `performance_profile_plot`.

Reply:
Done.